samples/CMakeLists.txt

```diff
@@ -108,7 +108,7 @@ add_test(
   COMMAND
     "${CMAKE_CTEST_COMMAND}"
             --build-and-test "${CMAKE_CURRENT_SOURCE_DIR}/videoDecode"
-                              "${CMAKE_CURRENT_BINARY_DIR}/videoDecode"
```

Comment:
This needs to be in the bin dir, else the git  source will have a whole bunch of files.